helpdesk_mgmt_ticket_start_date/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,19 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models, api
+
+import datetime
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    start_date = fields.Datetime()
```

Comment:
@ChristianSantamaria  One last suggested change, in order to log modifications:
```suggestion
    start_date = fields.Datetime(track_visibility="onchange")
```